lib/api/github.js

```diff
@@ -583,6 +585,7 @@ async function createPr(branchName, title, body, useDefaultBranch) {
     },
   })).body;
   pr.displayNumber = `Pull Request #${pr.number}`;
+  await addLabels(pr.number);
```

Comment:
Since this is the Github side, don't we need to pass labels here?

Reply:
yes!